types/govuk-frontend/dist/govuk/errors/index.d.ts

```diff
@@ -19,7 +19,7 @@ import { type ComponentWithModuleName } from "../common/index.js";
  * }
  * ```
  */
-export class GOVUKFrontendError extends Error {
+export abstract class GOVUKFrontendError extends Error {
```

Comment:
**question** We're using `@virtual` because of [Typedoc support for it rather than `@abstract`](https://github.com/alphagov/govuk-frontend/pull/5238/commits/d211124706c1328ef0ea576dcaf705d9d7001f71). Looks like [they've recently started supporting `@abstract` as well](https://github.com/TypeStrong/typedoc/issues/2692), would that make a difference for the type generation if we swapped back to `@abstract`?

Reply:
Ah yes, I manually added ~`export`~ `export abstract` as the JSDoc wasn't picked up

It probably won't make a difference but worth a try (it might be supported one day)

* https://github.com/microsoft/TypeScript/issues/17227